projects/fw_103/src/main.c

```diff
@@ -0,0 +1,50 @@
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "wait.h"
+
+static void button_callback(const GpioAddress *address, void *context) {
+  GpioAddress *adc_addr = context;
+  uint16_t adc_reading = 0;
+  adc_read_converted_pin(*adc_addr, &adc_reading);
+
+  LOG_DEBUG("A6 ADC converted reading: %d\n", adc_reading);
+}
+
+int main(void) {
+  interrupt_init();
+  gpio_it_init();
+  gpio_init();
+
+  GpioAddress adc_addr = { .port = GPIO_PORT_A, .pin = 6 };
```

Comment:
Just a small formatting thing. We normally add a comma at the end so that it is formated better
(ie just add a comma like this ... `.pin = 6,` so that the code will be formated properly when we run make format

Reply:
Same goes for the structs below